io.github.Adda0.Stardrop.yaml

```diff
@@ -0,0 +1,76 @@
+app-id: io.github.Adda0.Stardrop
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet7
+build-options:
+    append-path: /usr/lib/sdk/dotnet7/bin
+    append-ld-library-path: /usr/lib/sdk/dotnet7/lib
+    env:
+        PKG_CONFIG_PATH: /app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet7/lib/pkgconfig
+modules:
+  # Taken from https://github.com/flathub/com.unity.UnityHub/blob/af171c1ee22994e7ef698004bd44ba7e8bc71cf0/openssl-1.1.yaml.
+  - name: openssl-1.1
```

Comment:
openssl 1.1 is no longer developed.

Reply:
Unfortunately, it is. It is a long-standing [issue](https://github.com/Floogen/Stardrop/issues?q=is%3Aissue+libssl) with Stardrop. I would like to update all the dependencies in order to not need libssl1.1 any more, too, but that is still WIP.